modules/host/ghaf_host_hardened_baseline

```diff
@@ -1640,16 +1774,9 @@ CONFIG_HID_GENERIC=y
 #
 # USB HID support
 #
-CONFIG_USB_HID=m
```

Comment:
[Same here as with earlier comment.](https://github.com/tiiuae/ghaf/pull/415/files#r1429646562)
To support hardened host with no USB, these 
https://github.com/tiiuae/ghaf/pull/415/files#diff-bc4e2d3616e697e47c5c01ea825ca521e9c6ad6ce46288131c064c18304ced97R1777-R1862
should be behind ghaf option.

Reply:
Noted